### ISSUE TITLE
[TFC beta] Spy Disguise and Double Shot glitch

### ISSUE THREAD

**Issue by APGRoboCop** [ValveSoftware/halflife#1824]
Hello I noticed that those glitches that were mentioned already near Summer 2013 are still remained on this SteamPipe TFC build as well on the latest beta builds.

We hope you can wipe those glitches before it is too late - as the Spy Disguise and Double Shot glitch can be triggers maybe only after mapchange, by the looks of it. Here is my TFC beta client version, server version and Metamods used:-

**TFC Client version:-**
```
Protocol version 48
Exe version 1.1.2.2/Stdio (tfc)
Exe build: 09:10:05 Jul  7 2017 (7561)
```

**TFC Server version:-**
```
Protocol version 48
Exe version 1.1.2.2/Stdio (tfc)
Exe build: 17:35:12 Jul  5 2017 (7559)
```

**MetaMod v1.21 Addons List:-**
```
00:45:13 meta list
00:45:13 Currently loaded plugins:
               description      stat pend  file              vers      src   load  unlod
          [ 1] Adminmod         RUN   -    admin_mm.so       v2.50.61  ini   Start ANY  
          [ 2] LogDaemon        RUN   -    logd_mm.so        v1.0.0.7  ini   ANY   Pause
          [ 3] FoxBot           RUN   -    foxbot_mm.so      v0.79     ini   Start Start
         3 plugins, 3 running
```

So basically its just the Spy Disguise not working and the double shot glitch that occurs where it can be spotted easily using SPAS Super Shotgun or using RPG.

**Comment by SamVanheer** [comment]
Valve is aware of both issues, i've notified Alfred multiple times of these issues in particular. They're not considered critical and won't be fixed.

These issues contain more information, as well as the fixes for both:
#1330 
#1070

**Comment by pizzahut2** [comment]
I guess they only fix security related issues now. But come on Valve, breaking a whole class *is* a critical bug. What good is a spy if he can't even disguise? The two mentioned bugs forced server owners to revert to the 2012 version of the tfc.dll / tfc.so library files in order to fix the broken game. If that's not a critical bug, then I don't know what is.

**Comment by APGRoboCop** [comment]
@pizzahut2 
Beats me I guess its only fair they are working on Source II engines as TFC is nearly 18 years old. I know I have been too busy trying to fix and recompile some old Metamod Addons for it to work better on HLDS SteamPipe servers, to avoid too much hassle.

But for those TFC bugs and that as well other possible HLDS bugs that I've reported 3-4 years ago, I hope they will resolve those simple problems. Beggers can't be choosers - especially that games like Dota 2, CSGO and Source II comes first, which seems understandable.

**Comment by swmpdg** [comment]
I consider the spy disguise bugs critical. TFC is my all-time favorite mod since 2000. I can't play TFC anymore because the spy disguise issue took all the fun out of the game for me. I am one of the few people who actually play "Spy Defense". Please fix this Valve. We don't want to see the game completely die because of this. We would fix it if we could.

**Comment by SamVanheer** [comment]
Both of the issues listed here have been fixed, so this issue can be closed.

**Comment by kisak-valve** [comment]
Closing as a duplicate of #1070 / #1330.
